changelogs/fragments/350-raise-error-on-option-duplication-in-term-string.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - parse_kev_term() - function parsed the term string with option duplication. Function now raises error on option duplication in the term string.
```

Comment:
```suggestion
  - hashi_vault lookup - duplicate term-string-based options do not raise an error, leading to unexpected behavior (https://github.com/ansible-collections/community.hashi_vault/issues/349).
```

Reply:
From [docs](https://docs.ansible.com/ansible/devel/community/development_process.html#creating-a-changelog-fragment):
"Use past tense to describe the problem and present tense to describe the fix."
Is it fine to use present tense to describe bug? Also changelog doesn't contain fix description.

Reply:
Thank you, you're correct! I had that wrong in my mind, let's leave your wording as is, but please do add the link back to the issue.

Reply:
I've added issue link. If you want we can change wording.

Reply:
Updated in https://github.com/ansible-collections/community.hashi_vault/pull/350/commits/37e738c93a5352eca7162122516aa21d1d3029b0